airflow/api_connexion/openapi/v1.yaml

```diff
@@ -425,7 +425,12 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/DAG'
+              type: object
```

Comment:
We should named components here. Otherwise there may be problems with generating classes for API client  

 I personally prefer to always use named components.